README.md

```diff
@@ -33,6 +33,7 @@ third_party_api_rate_limit_bucket = SimpleTokenBucket(
     name="third_party_api",
     bucket_size=3,
     interval=10,
+    backend=RedisBackend(<<REDIS-CLIENT-OBJECT>>)
```

Comment:
I think we should use the [`from_url` classmethod](https://simple-token-bucket.readthedocs.io/en/latest/api.html#simple_token_bucket.backends.redis.RedisBackend.from_url) here.

```suggestion
    backend=RedisBackend.from_url("redis://localhost/0")
```

Reply:
Changed. I also put 6379 port just to be more clear about it